package.json

```diff
@@ -57,8 +57,6 @@
     "@qavajs/memory": "^1.0.0",
     "chai": "^4.3.6",
     "fs-extra": "^10.1.0",
-    "jsonpath": "^1.1.1",
-    "node-fetch": "^3.2.6",
-    "webdriverio": "^7.20.5"
+    "node-fetch": "^3.2.6"
```

Comment:
pls move @qavajs/memory and node-fetch to dependencies

Reply:
chai too

Reply:
discussed